tests/mime/message/part_test.py

```diff
@@ -620,3 +620,9 @@ def test_encode_transfer_encoding():
     encoded_body = encode_transfer_encoding('base64', body)
     # according to  RFC 5322 line "SHOULD be no more than 78 characters"
     assert_less(max([len(l) for l in encoded_body.splitlines()]), 79)
+
+
+def test_broke_base64():
```

Comment:
This checks only one case. I would suggest adding unit tests for `_decode_base64` function that cover all corruption cases.